src/Compilers/Core/Portable/Optional.cs

```diff
@@ -56,5 +56,15 @@ public T Value
         {
             return new Optional<T>(value);
         }
+
+        /// <summary>
+        /// Returns a string representation of the optional value.
+        /// </summary>
+        public override string ToString()
+        {
+            return _hasValue
+                ? _value.ToString()
```

Comment:
>  the doc comment says "Represents a value type that can be assigned null."

Probably should change the doc comment then given they didn't bother to constrain `T` to `struct`. :smile:

> Are we concerned about ToString returning null?

Not too much. I'm fine with normalizing it here.